locales/th-TH/translation.json

```
@@ -92,65 +92,65 @@
         "current-day": "ไปที่วันปัจจุบัน",
         "previous-day": "เมื่อวานนี้",
         "next-day": "พรุ่งนี้",
-        "time-to-leave": "Time to Leave",
+        "time-to-leave": "เวลาเลิกงาน",
         "leave-by": "ควรเลิกงาน:",
-        "day-done-balance": "All done for today. Balance of the day:",
-        "month-balance": "Month Balance",
-        "month-balance-title": "Balance up until today for this month. A positive balance means extra hours you don't need to work today (or the rest of the month).",
-        "overall-balance": "Overall Balance",
-        "overall-balance-title": "Overall balance until end of the month or current day",
-        "not-a-working-day": "Not a working day",
+        "day-done-balance": "วันนี้เรียบร้อยแล้ว จำนวนชั่วโมงส่วนเกินของวันนี้:",
+        "month-balance": "จำนวนชั่วโมงส่วนเกินในเดือนนี้",
+        "month-balance-title": "จำนวนชั่วโมงส่วนเกินจนถึงวันนี้สำหรับเดือนนี้ จำนวนที่เป็นบวกหมายถึงชั่วโมงส่วนเกินที่คุณไม่ต้องทำงานสำหรับวันนี้ (หรือสำหรับวันที่เหลือของเดือน)",
+        "overall-balance": "จำนวนชั่วโมงส่วนเกิน",
+        "overall-balance-title": "จำนวนช่วงโมงส่วนเกินจนถึงสิ้นเดือนหรือวันนี้",
+        "not-a-working-day": "ไม่ใช่วันทำงาน",
         "waived-day": "วันหยุด",
-        "day-total": "Day Total",
-        "remove-entry": "Remove entry",
-        "entry-removal-confirmation": "Are you sure you want to remove the last entry row?",
+        "day-total": "วันทั้งหมด",
+        "remove-entry": "ลบรายการ",
+        "entry-removal-confirmation": "คุณมั่นใจหรือไม่ที่จะลบรายการล่าสุด?",
         "yes": "ใช่",
         "no": "ไม่",
-        "entry": "Entry"
+        "entry": "รายการ"
     },
     "$FlexibleMonthCalendar": {
         "switch-view": "เปลี่ยนมุมมอง",
-        "current-month": "Go to Current Month",
-        "previous-month": "Previous Month",
-        "next-month": "Next Month",
-        "time-to-leave": "Time to Leave",
-        "day": "Day",
-        "total": "Total",
-        "leave-by": "Based on the time you arrived today, you should leave by",
-        "day-done-balance": "All done for today. Balance of the day:",
-        "last-day-balance": "Last day used for balance",
+        "current-month": "ไปยังเดือนปัจจุบัน",
+        "previous-month": "เดือนที่แล้ว",
+        "next-month": "เดือนหน้า",
+        "time-to-leave": "เวลาเลิกงาน",
```

Time to Leave is the name of the App, because of this we are not translating it. :)
```suggestion
        "time-to-leave": "Time to Leave",
```